### PR TITLE
fix(build): update node-sass to beta, add npm-shrinkwrap, use proper ver...

### PR DESCRIPTION
**Pull request by RobertoUa**
...sions of packages, clean 'install-all' task

### CODE REVIEW

**Comment by RobertoUa** [comment]
fixes #1

File: package.json [hunk 1]
```diff
@@ -18,24 +18,24 @@
...
+    "gulp": "^3.8",
     "gulp-react": "^3.0.1",
-    "gulp-sass": "^1.3.3",
+    "gulp-sass": "^2.0.0-alpha.1",
     "gulp-typescript": "^2.6.0",
-    "node-sass": "latest",
-    "react-tools": "latest",
-    "typescript": "latest"
+    "node-sass": "^3.0.0-beta.7",
+    "react": "^0.13",
```
Comment (RobertoUa) [inline package.json]: didn't notice that react-tools has changed to just `react`
Now it's in both deps lists

File: package.json [hunk 2]
```diff
@@ -18,24 +18,24 @@
...
+    "typescript": "^1.5.0-alpha"
   },
   "scripts": {
-    "install-all": "rm -rf node_modules && npm update && npm run install-native && npm install && npm run cleanup",
```
Comment (shockone) [inline package.json]: Why have your removed the install parts?

**Comment by shockone** [comment]
@RobertoUa, it should work with the current master.

**Pull request closed without merging**